tests/unit/lms/views/application_instance_test.py

```diff
@@ -70,6 +39,10 @@ def test_it_sets_settings(
                 }
             ),
         )
+        assert result == {
+            "consumer_key": application_instance_service.create_application_instance.return_value.consumer_key,
+            "shared_secret": application_instance_service.create_application_instance.return_value.shared_secret,
+        }
```

Comment:
Popping out `application_instance_service.create_application_instance.return_value` as `created_ai` or something might make this clearer.